src/scss/templates/_sweepstakes.scss

```diff
@@ -0,0 +1,54 @@
+.sweepstakes {
+
+    &__country-label {
+        top: 50%;
+        left: 50%;
+        transform: translate(-50%, -50%);
+
+        &:after {
+            font-size: calc-em(10, 16);
+            margin-left: calc-em(10, 16);
+        }
+    }
+
+    &__checkbox {
+        width: calc-em(19, 16);
+        height: calc-em(19, 16);
+        border-radius: calc-em(3, 16);
+        color: $primary;
```

Comment:
if our checkbox is always like this we don't need a custom one here, please check other checkbox uses across the website, if we only have this we can rather update Form components

Reply:
checked across all page, there is checkbox on account register & dashboard, its also using custom div. The only difference is bgcolor reverse, I think can use utility class for that. So I created checkbox form component